### PR TITLE
Qwiklabs HTML Output (CLaaT)

### PR DESCRIPTION
**Pull request by supersam654**
## Changes:

* Every link to the `googlecodelabs` version of CLaaT has been replaced with this fork.
* The CLaaT CLI can now ingest GDocs from Team Drives.
*  `render/qwiklabs.go` is just a copied version of the HTML renderer with a few names changed so there are no naming collisions. Nothing in there is terribly interesting (in fact, I'd call it less interesting than the existing HTML renderer).

### CODE REVIEW

**Comment by supersam654** [comment]
[CLAAT: Bolded spaces become underscores](https://trello.com/c/2Rj6Dhvd/1089-claat-bolded-spaces-become-underscores)

**Comment by jgan** [comment]
Preliminary LGTM

Approval pending getting this deployed and tested with old and new versions of the website code.

**Comment by supersam654** [comment]
[Use claat HTML export instead of Markdown](https://trello.com/c/anWlxDif/1268-use-claat-html-export-instead-of-markdown)

File: claat/fetch.go [hunk 1]
```diff
@@ -197,7 +197,7 @@ func fetchDriveFile(id string, nometa bool) (*resource, error) {
 		return &resource{body: res.Body, typ: srcGoogleDoc}, nil
 	}
 
-	u := fmt.Sprintf("%s/files/%s?fields=id,mimeType,modifiedTime", driveAPI, id)
+	u := fmt.Sprintf("%s/files/%s?fields=id,mimeType,modifiedTime&supportsTeamDrives=true", driveAPI, id)
```
Comment (jgan) [inline claat/fetch.go:200]: :+1:

**Review by jgan — APPROVED** [review]
LGTM

**Pull request merged**